src/store.h

```diff
@@ -54,6 +54,11 @@ class MetadataStore {
           version(other.version), schema_name(other.schema_name),
           is_deleted(other.is_deleted), collected_at(other.collected_at),
           metadata(std::move(other.metadata)), ignore(other.ignore) {}
+    Metadata(const Metadata& other)
```

Comment:
This is the same as `other.Clone()`. Why do you need this?

Reply:
removed